### PR TITLE
Add build tags to generated packr files

### LINKED ISSUE #153: Feature Request: Optionally have build tags in generated files
First, Thanks for packr 😄 it is great.

As title. Right now I append the build tags after running packr, and it would be nice to have that all as one step.

### PR DESCRIPTION
**Pull request by techknowlogick**
Closes #153

### CODE REVIEW

**Comment by markbates** [comment]
Currently travis is having problems with Go builds, so once that's cleared up and the tests pass, I'll merge this in. Thanks.

**Comment by markbates** [comment]
@techknowlogick it looks like you can't use `-` in a build tag:

```
packrd/packed-packr.go:1: invalid non-alphanumeric build constraint: !skip-packr
```

File: v2/jam/store/disk_tmpl.go [hunk 1]
```diff
@@ -1,6 +1,7 @@
 package store
 
-const diskGlobalTmpl = `// Code generated by github.com/gobuffalo/packr/v2. DO NOT EDIT.
+const diskGlobalTmpl = `// +build !skip-packr
```
Comment (markbates) [inline v2/jam/store/disk_tmpl.go]:
```suggestion
const diskGlobalTmpl = `// +build !skippackr
```

File: v2/jam/store/disk_tmpl.go [hunk 2]
```diff
@@ -31,7 +32,8 @@ var _ = func() error {
 }()
 `
 
-const diskImportTmpl = `// Code generated by github.com/gobuffalo/packr/v2. DO NOT EDIT.
+const diskImportTmpl = `// +build !skip-packr
```
Comment (markbates) [inline v2/jam/store/disk_tmpl.go]:
```suggestion
const diskImportTmpl = `// +build !skippackr
```

Comment (techknowlogick) [reply v2/jam/store/disk_tmpl.go]: Thanks for the feedback. I've updated with your changes 😄

**Comment by markbates** [comment]
Thanks!

**Pull request merged**